extensions/geode-modules-test/build.gradle

```diff
@@ -27,7 +27,10 @@ dependencies {
     exclude group: 'org.apache.tomcat'
   }
   compile project(':geode-junit')
-  compile 'org.httpunit:httpunit:' + project.'httpunit.version'
+  compile ('org.httpunit:httpunit:' + project.'httpunit.version') {
```

Comment:
If we upgrade the version of httpunit will it fix the terces issue (so we don't need a separate dependency)?

Reply:
httpunit is a stale component now. the last update is 2012